tests/openssl_sgx_tcrypto_compat/enc/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../openssl_sgx_tcrypto_compat.edl)
+
+add_custom_command(
+  OUTPUT openssl_sgx_tcrypto_compat_t.h openssl_sgx_tcrypto_compat_t.c
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND
+    edger8r --trusted ${EDL_FILE} --search-path ${PROJECT_SOURCE_DIR}/include
+    ${DEFINE_OE_SGX} --search-path ${CMAKE_CURRENT_SOURCE_DIR})
+
+add_enclave(
+  TARGET
+  openssl_sgx_tcrypto_compat_enc
+  CXX
+  UUID
+  90c020e8-e5dd-11ea-adc1-0242ac120002
```

Comment:
If you specify CRYPTO_LIB openssl, the enclave should automatically link against `openssl`, include the OpenSSL headers, and initialize the rdrand engine. Can you try it and simplify the code?

See:
https://github.com/openenclave/openenclave/blob/4aade5cde8c38a2d681353f753f6c84d97c8c084/tests/attestation_plugin_cert/enc/CMakeLists.txt#L35

Reply:
I took the `CRYPTO_LIB` `openssl` and it doesn't need link to openssl static lib explicitly any more. 
But I don't see the OpenSSL header files folder, neither under the build folder nor the original source,  added into including path.